<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEditor</class>
 <widget class="QWidget" name="EditEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lb_start">
       <property name="text">
        <string>Bit Start:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_start"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lb_length">
       <property name="text">
        <string>Bit Length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_length">
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pte_bits"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
